doc/src/user_guide.rst

```diff
@@ -712,1034 +560,995 @@ Example::
 
-Example --- 2D Incompressible Cylinder Flow
-===========================================
+1. ``nsteps`` --- calculate the integral every ``nsteps`` time steps:
 
-Proceed with the following steps to run a serial 2D incompressible cylinder
-flow simulation on a mixed unstructured mesh:
+    *int*
 
-1. Create a working directory called ``inc_cylinder_2d/``
+2. ``file`` --- output file path; should the file already exist it
+   will be appended to:
 
-2. Copy the configuration file
-   ``PyFR/examples/inc_cylinder_2d/inc_cylinder_2d.ini`` into
-   ``inc_cylinder_2d/``
+    *string*
 
-3. Copy the compressed `Gmsh <http:http://geuz.org/gmsh/>`_ mesh file
-   ``PyFR/examples/inc_cylinder_2d/inc_cylinder_2d.msh.gz`` into
-   ``inc_cylinder_2d/``
+3. ``header`` --- if to output a header row or not:
 
-4. Unzip the file and run pyfr to covert the `Gmsh <http:http://geuz.org/gmsh/>`_
-   mesh file into a PyFR mesh file called ``inc_cylinder_2d.pyfrm``::
+    *boolean*
 
-        zcat inc_cylinder_2d.msh.gz | pyfr import -tgmsh - inc_cylinder_2d.pyfrm
+4. ``int``-*name* --- expression to integrate, written as a function of
+   the primitive variables and gradients thereof; multiple expressions,
+   each with their own *name*, may be specified:
 
-5. Run pyfr to solve the incompressible Navier-Stokes equations on the mesh,
-   generating a series of PyFR solution files called
-   ``inc_cylinder_2d-*.pyfrs``::
+    *string*
 
-        pyfr run -b cuda -p inc_cylinder_2d.pyfrm inc_cylinder_2d.ini
+Example::
 
-6. Run pyfr on the solution file ``inc_cylinder_2d-75.00.pyfrs``
-   converting it into an unstructured VTK file called
-   ``inc_cylinder_2d-75.00.vtu``. Note that in order to visualise the
-   high-order data, each high-order element is sub-divided into smaller
-   linear elements. The level of sub-division is controlled by the
-   integer at the end of the command::
+    [soln-plugin-integrate]
+    nsteps = 50
+    file = integral.csv
+    header = true
+    vor1 = (grad_w_y - grad_v_z)
+    vor2 = (grad_u_z - grad_w_x)
+    vor3 = (grad_v_x - grad_u_y)
 
-        pyfr export inc_cylinder_2d.pyfrm inc_cylinder_2d-75.00.pyfrs inc_cylinder_2d-75.00.vtu -d 4
+    int-E = rho*(u*u + v*v + w*w)
+    int-enst = rho*(%(vor1)s*%(vor1)s + %(vor2)s*%(vor2)s + %(vor3)s*%(vor3)s)
 
-7. Visualise the unstructured VTK file in `Paraview
-   <http://www.paraview.org/>`_
+Additional Information
+----------------------
 
-.. figure:: ../fig/inc_cylinder_2d/inc_cylinder_2d.png
-   :width: 450px
-   :figwidth: 450px
-   :alt: couette flow
-   :align: center
+The ini file format is very versatile. A feature that can be useful in
+defining initial conditions is the substitution feature and this is
+demonstrated in the :ref:`integrate-plugin` example.
 
-   Colour map of velocity magnitude distribution at 75 time units.
+To prevent situations where you have solutions files for unknown
+configurations, the contents of the ini file is added as an attribute
```

Comment:
Should this ini be .ini ? We seem to use a mix of ini .ini ``.ini`` and INI. Maybe quick check for consistency?